api/com_res/main.py

```diff
@@ -42,84 +33,8 @@
     allow_headers=["*"],
 )
 
-app.include_router(
-    argo_router,
-    # prefix="/auth/cuahsi",
-    tags=["argo"],
-)
-
-app.include_router(
-    utilities_router,
-    tags=["utilities"],
-)
-
-app.include_router(
-    access_control_router,
-    # prefix="/auth/cuahsi",
-    tags=["minio"],
-)
-
-app.include_router(
-    storage_router,
-    # prefix="/auth/cuahsi",
-    tags=["minio"],
-)
```

Comment:
You don't think this app will ever use CUAHSI SSO?

Reply:
@devincowan I removed the API endpoints because they are not being used in this application (and aren't configured). I originally had them commented out, but figured that it would be better to just remove them. I didn't remove any of their supporting code, so that we can use them in the future.